### DISCUSSION TITLE
Full xbox control - Lego 42146 Liebherr LR13000

### DISCUSSION THREAD

**Discussion by vanczakp** [pybricks/support#1866] [Show and tell]
Hello All,

Please welcome here my version of LR13000 xbox control.
Works with original built Lego 42146. In case you extended the booms, cables, then do not calibrate unless you understand how the program works. Created in beta enviroment.
It features:
- Graphic user manual
- full analogue control of all drives,
- one or two stick tank control of tracks, 
- end position stop of booms and hoist (when calibration is done), 
- weight overload warning
- 5 speed and acceleration (independet for bottom and top drives) profiles
- 7 different control profiles

[LR13000_full_control.zip](https://github.com/user-attachments/files/17268913/LR13000_full_control.zip)

**Comment by BertLindeman** [comment]
Hi Péter,

Pity I have taken down my LR1300.
I'll have to rebuild again, and that is fun again. 😄 

Thank you for sharing your extensive work.

Bert

**Reply by vanczakp** [reply]
Welcome! Hope you will enjoy it! Please give me feedback/ideas to improve when aou tried it!
